docs/api/browser-window.md

```diff
@@ -1478,7 +1478,21 @@ removed in future Electron releases.
 
 #### `win.getBrowserView()` _Experimental_
 
-Returns `BrowserView | null` - an attached BrowserView. Returns `null` if none is attached.
+Returns `BrowserView | null` - an BrowserView attached by setBrowserView. Returns `null` if none is attached.
+
+#### `win.addBrowserView(browserView)` _Experimental_
+
+* `browserView` [BrowserView](browser-view.md)
+
+Replacement API for setBrowserView supporting work with multi browser views.
```

Comment:
I think we should have a clear definition on the behavior when using `setBrowserView` and `addBrowserView` together, and have it documented. I suggest:

1. Make `setBrowserView` an alias for `removeAllBrowserViews() && addBrowserView(view)`.
2. Make `getBrowserView` throw if there are multiple browser views, and return `getBrowserViews()[0]` as fallback.

Reply:
documentation updated